### ISSUE TITLE
Commands are not working for CheckBox and RadioButton control

### ISSUE THREAD

**Issue by wieslawsoltes** [Perspex/Perspex#526]
Code to reproduce:
```XAML
<CheckBox Command="{Binding SomeCommand}" CommandParameter="SomeParam"/>
<RadioButton Command="{Binding SomeCommand}" CommandParameter="SomeParam"/>
```

**Comment by jkoritzinsky** [comment]
@grokys Since this has been fixed, can we close the issue?

**Comment by grokys** [comment]
Yep! Closed by #527 #528
